src/utils/google_drive.py

```diff
@@ -68,6 +66,7 @@ def _get_credentials(self):
             flow = client.flow_from_clientsecrets(CLIENT_SECRET_FILE, SCOPES)
             flow.user_agent = self.app_name
             try:
+                import argparse

```

Comment:
So said Python 2.6 compatibility (but I'm not sure if script really works in Python 2.6 environment).